### PR TITLE
Kitchen Resprites

### PR DESCRIPTION
**Pull request by alsoandanswer**
dreamseeker_qiQE9bs2i4
dreamseeker_xN1dibd6DI

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by atteria** [comment]
The chopping board resprite no longer looks like a chopping board, and instead is just the ugly tg wood texture stretched out, which Aurora doesn't need more of. This is the opposite of a sprite that clearly communicates what it is supposed to be.

The grill resprite is using the plastic palette for some reason, when it's a grill. The pasted tg kitchen dials look clumsy, inconsistent with the kitchen resprite from this year, and not like an actual grill. If you want to paste dials, use the knobs on my stove sprites.

Please reference what a thing is intended to look like rather than tg 50s retrofuture sprites.

**Comment by alsoandanswer** [comment]
Alright, I've made some changes, hopefully this will fix them

> The chopping board resprite no longer looks like a chopping board, and instead is just the ugly tg wood texture stretched out, which Aurora doesn't need more of. This is the opposite of a sprite that clearly communicates what it is supposed to be.

dreammaker_6buM2cQNaz

> The grill resprite is using the plastic palette for some reason, when it's a grill. The pasted tg kitchen dials look clumsy, inconsistent with the kitchen resprite from this year, and not like an actual grill. If you want to paste dials, use the knobs on my stove sprites.

dreammaker_jTuVFUwWvn
Here's the new kitchen appliance sprites - I've tweaked them accordingly.
Only real point of contention for me is the knobs, which I've changed, the current ones are too unreadable.
(ironically, using the current knobs is closer to what /tg/ uses currently)
dreammaker_0nmC5QBh67

> The oven dish no longer looks like a commercial bread pan, and confusingly like a tray.
>

dreammaker_FtGYRsbHMl

**Comment by niennab** [comment]
Didn't we already talk about no wholesale respriting of recent assets based on TG sprites?

**Comment by alsoandanswer** [comment]
They've been redone again.

**Comment by Zelmana** [comment]
I like some of these. My only complaint is the huge wine bottle. It's as tall as a character sprite! The largest bottles, Imperials, are pretty huge but IDK if they're that big....

**Comment by alsoandanswer** [comment]
> I like some of these. My only complaint is the huge wine bottle. It's as tall as a character sprite! The largest bottles, Imperials, are pretty huge but IDK if they're that big....

The champagne bottle was resprited in a different PR that was already merged

File: code/modules/cooking/machinery/cooking_machines/container.dm [hunk 1]
```diff
@@ -17,9 +17,27 @@
 		/obj/item/stack/rods,
 		/obj/item/organ/internal/brain
 		)
+	drop_sound = 'sound/items/drop/metal_pot.ogg'
+	pickup_sound = 'sound/items/pickup/metal_pot.ogg'
 	var/appliancetype // Bitfield, uses the same as appliances
 	w_class = ITEMSIZE_NORMAL
 
+/obj/item/reagent_containers/cooking_container/on_reagent_change()
+	..()
```
Comment (FluffyGhoster) [inline code/modules/cooking/machinery/cooking_machines/container.dm:26]:
```suggestion
	. = ..()
```

File: code/modules/cooking/machinery/cooking_machines/container.dm [hunk 2]
```diff
@@ -17,9 +17,27 @@
...
+	update_icon()
+
+/obj/item/reagent_containers/cooking_container/pickup(mob/user)
+	..()
```
Comment (FluffyGhoster) [inline code/modules/cooking/machinery/cooking_machines/container.dm:30]:
```suggestion
	. = ..()
```

File: code/modules/cooking/machinery/cooking_machines/container.dm [hunk 3]
```diff
@@ -17,9 +17,27 @@
...
+	update_icon()
+
+/obj/item/reagent_containers/cooking_container/dropped(mob/user)
+	..()
```
Comment (FluffyGhoster) [inline code/modules/cooking/machinery/cooking_machines/container.dm:34]:
```suggestion
	. = ..()
```

File: code/modules/cooking/machinery/cooking_machines/container.dm [hunk 4]
```diff
@@ -17,9 +17,27 @@
...
+	update_icon()
+
+/obj/item/reagent_containers/cooking_container/attack_hand()
+	..()
```
Comment (FluffyGhoster) [inline code/modules/cooking/machinery/cooking_machines/container.dm:38]:
```suggestion
	. = ..()
```

**Review by niennab — CHANGES_REQUESTED** [review]
grill_on and cereal_on have frames of other appliances. Refer to IconDiffBot2: icons/obj/cooking_machines.dmi (0)

I still don't agree with overhauling and downgrading atteria's work but can we at least get rid of the weirdly huge dials?

**Review by NonQueueingMatt — DISMISSED** [review]
I don't like this tendency towards retrofuture beige for appliance colour - we're not retrofuture. Remove it from the machines that use it, use pure white or black or whatever else doesn't look like a 50s appliance.

**Comment by alsoandanswer** [comment]
> I don't like this tendency towards retrofuture beige for appliance colour - we're not retrofuture. Remove it from the machines that use it, use pure white or black or whatever else doesn't look like a 50s appliance.

Addressed, they all are using a metallic shine. There is no longer any beige at all

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**